package.json

```diff
@@ -20,7 +21,7 @@
         "type": "git"
     },
     "devDependencies": {
-        "arches-dev-dependencies": "archesproject/arches-dev-dependencies#dev/7.6.x"
+        "arches-dev-dependencies": "archesproject/arches-dev-dependencies#jtw/prettier"
```

Comment:
Of course I'll revert this once https://github.com/archesproject/arches-dev-dependencies/pull/15 is merged.

Reply:
To test the pre-commit hook in a new project, you'll want to add this change to the project template file before creating it (or manually edit afterward and nuke/reinstall yarn)